d3a_api_client/redis.py

```diff
@@ -9,7 +10,7 @@
 from d3a_api_client.constants import MAX_WORKER_THREADS
 
 root_logger = logging.getLogger()
-root_logger.setLevel(logging.ERROR)
+root_logger.setLevel(logging.INFO)
```

Comment:
Any particular reason to change the log level?

Reply:
No, this is a remnant of testing. Will change it back.